readabilitySAX.js

```diff
@@ -150,7 +150,7 @@ var tagsToSkip = {__proto__:null,aside:true,footer:true,head:true,label:true,nav
     divToPElements = ["a","blockquote","dl","img","ol","p","pre","table","ul"],
     okayIfEmpty = ["audio","embed","iframe","img","object","video"],
 
-    re_videos = /http:\/\/(?:www\.)?(?:youtube|vimeo)\.com/,
+    re_videos = /https?:\/\/(?:www\.)?(?:youtube|vimeo)\.(?:com)/,
```

Comment:
Yep. Left their mistakenly, added that for slideshare.net - but later removed that site from regular expression.